components/atomic/Flag/index.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+function Flag({ className, children, color }) {
+  return (
+    <Styled.Flag className={className}>
+      <Styled.OffsetWrapper>
+        <Styled.FlagBody $backgroundColor={color}>{children}</Styled.FlagBody>
+        <Styled.FlagTail
+          xmlns="http://www.w3.org/2000/svg"
+          role="presentation"
+          viewBox="0 0 100 24.551"
+        >
+          <defs>
+            <filter id="Subtraction_6" x="0" y="0" filterUnits="userSpaceOnUse">
+              {/* eslint-disable-next-line react/no-unknown-property */}
+              <feOffset dy="1" input="SourceAlpha" />
+              <feGaussianBlur result="blur" />
+              <feFlood floodOpacity="0.059" />
+              <feComposite operator="in" in2="blur" />
+              <feComposite in="SourceGraphic" />
+            </filter>
+          </defs>
+          <g transform="matrix(1, 0, 0, 1, 0, 0)" filter="url(#Subtraction_6)">
+            <path
+              id="Subtraction_6-2"
+              data-name="Subtraction 6"
+              d="M100,120h0L50,96.587,0,120V96.449H100V120Z"
+              transform="translate(0 -96.45)"
+              fill={color}
+            />
+          </g>
+        </Styled.FlagTail>
```

Comment:
This is a styled svg.  I thought about putting it alongside the other svg icons, but felt so bespoke to this component (which in turn didn't feel like it should be alongside the svgs because it's slightly more so than that) that I just inlined it here.